server/lib/cache-api-user-has-run-as.ts

```diff
@@ -51,28 +51,46 @@ export const APIUserAllowRunAs = {
         {},
         { apiHostID: apiId }
       );
-      const statusUserAllowRunAs = response.data.data.affected_items[0].allow_run_as ? API_USER_STATUS_RUN_AS.ENABLED : API_USER_STATUS_RUN_AS.NOT_ALLOWED;
+      const statusUserAllowRunAs = response.data.data.affected_items[0].allow_run_as ? API_USER_STATUS_RUN_AS.ENABLED : API_USER_STATUS_RUN_AS.INTERFACE_DISABLED;
 
       // Cache the run_as for the API user
       CacheInMemoryAPIUserAllowRunAs.set(apiId, api.username, statusUserAllowRunAs);
       return statusUserAllowRunAs;
     }catch(error){
       log('APIUserAllowRunAs:check', error.message || error);
-      return API_USER_STATUS_RUN_AS.DISABLED;
+      return API_USER_STATUS_RUN_AS.ALL_DISABLED;
     }
   },
   async canUse(apiId: string): Promise<number | never>{
     const ApiUserCanUseStatus = await APIUserAllowRunAs.check(apiId);
-    if(ApiUserCanUseStatus === API_USER_STATUS_RUN_AS.NOT_ALLOWED){
+    if(ApiUserCanUseStatus === API_USER_STATUS_RUN_AS.HOST_DISABLED){
```

Comment:
This should be `INTERFACE_DISABLED`. With `HOST_DISABLED` I can't pass the health check even if am logged with the `wazuh-wui` user and the `run_us` false.
![Captura de pantalla 2021-03-18 a las 10 39 08](https://user-images.githubusercontent.com/46006326/111612248-9444ef80-87dd-11eb-9310-2fd211d5e962.png)
![Captura de pantalla 2021-03-18 a las 10 39 38](https://user-images.githubusercontent.com/46006326/111612255-95761c80-87dd-11eb-9599-b35ef45b63b9.png)

Reply:
Hi Pablo,
I doubled checked it and fixed it. Thanks!